### PR TITLE
Fix paths

### PR DESCRIPTION
**Pull request by injust**
ff849d1 changed the path of decode.h, this fixes it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ Installs the libraries and the necessary include files.
 The include files are placed in a brotli subdir under the install prefix so
 include them like
 
-	#include <brotli/decode.h>
-	#include <brotli/encode.h>
+	#include <brotli/include/brotli/decode.h>
+	#include <brotli/include/brotli/encode.h>
```
Comment (bagder) [inline README.md]: After `make install`, the include files are not in that path. They're installed in $prefix/include/brotli/

**Comment by bagder** [comment]
If you remove the README change I can merge the correct configure fix.

**Comment by injust** [comment]
Fixed it, sorry about that slip-up.

**Comment by bagder** [comment]
thanks!

**Pull request merged**